src/main/java/com/avispl/symphony/dal/communicator/barco/clickshare/BarcoClickShareCommunicator.java

```diff
@@ -0,0 +1,524 @@
+                    break;
+                case STANDBY_TIMEOUT_NAME:
+                    patchDeviceMode(POWER_MANAGEMENT,"standbyTimeout", value, property);
+                    break;
+                case VIDEO_MODE_NAME:
+                    patchDeviceMode(CONFIGURATION_VIDEO,"mode", value, property);
+                    break;
+                case MIRACAST_NAME:
+                    patchDeviceMode(FEATURES_MIRACAST, "enabled", "0".equals(value) ? "false" : "true", property);
+                    break;
+                case GOOGLECAST_NAME:
+                    patchDeviceMode(FEATURES_GOOGLECAST, "enabled", "0".equals(value) ? "false" : "true", property);
+                    break;
+                case AIRPLAY_NAME:
+                    patchDeviceMode(FEATURES_AIRPLAY, "enabled", "0".equals(value) ? "false" : "true", property);
+                    break;
+                case BLACKBOARD_SAVING_NAME:
+                    patchDeviceMode(FEATURES_BLACKBOARD, "savingEnabled", "0".equals(value) ? "false" : "true", property);
+                    break;
+                case AUDIO_NAME:
+                    patchDeviceMode(CONFIGURATION_AUDIO, "enabled", "0".equals(value) ? "false" : "true", property);
+                    break;
+                case AUDIO_OUTPUT_NAME:
+                    patchDeviceMode(CONFIGURATION_AUDIO, "output", value, property);
+                    break;
+                case LANGUAGE_NAME:
+                    patchDeviceMode(PERSONALIZATION, "language", value, property);
+                    break;
+                case WELCOME_MESSAGE_NAME:
+                    patchDeviceMode(PERSONALIZATION, "welcomeMessage", value, property);
+                    break;
+                case REBOOT_NAME:
+                    requestReboot();
+                    break;
+                case STANDBY_NAME:
+                    requestStandby();
+                    break;
+                default:
+                    if(logger.isWarnEnabled()){
+                        logger.warn(String.format("Operation %s with value %s is not supported.", property, value));
+                    }
+                    break;
+            }
+        } finally {
+            controlOperationsLock.unlock();
+        }
+    }
+
+    @Override
+    public void controlProperties(List<ControllableProperty> list) throws Exception {
+        if (CollectionUtils.isEmpty(list)) {
+            throw new IllegalArgumentException("Controllable properties cannot be null or empty");
+        }
+
+        for(ControllableProperty controllableProperty: list){
+            controlProperty(controllableProperty);
+        }
+    }
+
+    @Override
+    protected void authenticate() throws Exception {
+    }
+
+    @Override
+    public List<Statistics> getMultipleStatistics() throws Exception {
+        ExtendedStatistics extendedStatistics = new ExtendedStatistics();
+
+        if(isValidControlCoolDown() && localStatistics != null){

```

Comment:
Not thread safe. The localStatistics is being accessed while not under lock - in all other places it is under lock.